ingredients/DECam/DRP.yaml

```diff
@@ -44,6 +44,7 @@ tasks:
       photo_calib_provider: "tract"
       background_provider: "input_summary"
       connections.photoCalibName: "jointcal"
+      connections.skyWcsName: "jointcal"
```

Comment:
Up in line 13, I think you should exclude `gbdesAstrometricFit` along with fgcm.